src/test/java/api/loans/RenewByIdTests.java

```diff
@@ -7,8 +7,22 @@
 import org.folio.circulation.support.http.client.Response;
 import org.folio.circulation.support.http.server.ValidationError;
 import org.hamcrest.Matcher;
+import org.junit.After;
+import org.junit.Before;
 
 public class RenewByIdTests extends RenewalAPITests {
+
+  @Before
+  public void setupLoanType() {
+    noteTypeFixture.generalNoteType();
+  }
```

Comment:
The name refers to a loan type yet the implementation refers to a note type, could this be renamed to remove this discrepancy?